notebook/services/contents/filemanager.py

```diff
@@ -544,7 +543,7 @@ def is_non_empty_dir(os_path):
                 # raises let us distinguish permission errors from other errors in
                 # code. So for now, just let them all get logged as server errors.
                 send2trash(os_path)
```

Comment:
You probably need to remove this call as well, I'd imagine

Reply:
Thanks for the hint!